codegen/runner/pre-steps.sh

```diff
@@ -58,9 +59,37 @@ end=`date +%s`
 runtime=$((end-start))
 echo "Compiled easyjson : +$runtime"
 
+go build -o $RESOLVE_THRIFT_BINARY $RESOLVE_THRIFT_FILE
+
+# find the modules that actually need JSON (un)marshallers
+target_dirs=""
+found_thrifts=""
+config_files=$(find ${CONFIG_DIR} -name "*-config.json" | sort)
```

Comment:
Did you resolve imports ?

If we import X in a http+json thrift file and embed it we need a json marshaller generated for x

Reply:
There is no need to resolve imports if the `"zanzibar.http.req.def"` annotation is not used, i.e., the struct that wraps all the method arguments is (un)marshalled. E.g., [foo.FooStruct](https://github.com/uber/zanzibar/blob/lu.ej/examples/example-gateway/idl/clients/bar/bar.thrift#L104) is imported from `examples/example-gateway/clients/foo/foo.thrift`, here [Bar_TooManyArgs_Args](https://github.com/uber/zanzibar/blob/lu.ej/examples/example-gateway/build/gen-code/clients/bar/bar/bar_toomanyargs.go#L14) is used and there is unwrapping of it, we don't need a standalone json marshaller for `foo.FooStruct`, because `foo.FooStruct` marshallers are already defined in [examples/example-gateway/build/gen-code/clients/bar/bar/bar_toomanyargs_easyjson.go](https://github.com/uber/zanzibar/blob/lu.ej/examples/example-gateway/build/gen-code/clients/bar/bar/bar_toomanyargs_easyjson.go#L448-L624).